Documentation/project-docs/how-to-track-changes.md

```diff
@@ -0,0 +1,164 @@
+from individual repos may not match the build version of Microsoft.NetCore.App 
+or an installer that aggregated them.
+
+## Our starting point
+
+In my case I had a git commit: [435a69a9aca037f348a09e00dffef8a0888682a7 
+in the coreclr repo](https://github.com/dotnet/coreclr/commit/435a69a9aca037f348a09e00dffef8a0888682a7) 
+and I wanted to know where this code had propagated to. Alternatively
+you may be starting from a particular package or installer and wondering
+what source it contains. Lets start exploring!
+
+
+## Where are the daily build NuGet packages published?
+
+For many of the dotnet repos such as CoreCLR the daily builds are posted to the 
+[dotnet-core](https://dotnet.myget.org/gallery/dotnet-core) feed. Other repos or 
+specific branches might publish elsewhere in which case hopefully you can find 
+that in the repo docs or by searching [the gallery](https://dotnet.myget.org/gallery) 
+for an appropriate sounding feed name.
+
+
+## What (intermediate) NuGet packages does a given repo produce?
+
+Whenever a dotnet repo publishes new builds it makes a commit in the 
+[dotnet/versions repo](https://github.com/dotnet/versions) indicating which 
+packages got published. For example during the night after I merged my 
+example CoreCLR change 
+[this commit](https://github.com/dotnet/versions/commit/cf8930fbe52e5eacf8ab0d7fb06f032d19cda5d5#diff-5f6099c37f777c410c4397b3f1e38870)
+was merged in dotnet/versions. Looking at 
+[Last_Build_Packages.txt](https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/master/Last_Build_Packages.txt) 
+which was modified in that commit you can see a list of NuGet packages that 
+CoreCLR repo publishes.
+
+
+## What git commit was a NuGet package built from (answer via MyGet)?
+
+Pick an arbitrary NuGet package produced by your repo of interest and look 
+for it on the MyGet [gallery](https://dotnet.myget.org/gallery). For example
+one of the [coreclr packages](https://dotnet.myget.org/feed/dotnet-core/package/nuget/runtime.win-x64.Microsoft.NETCore.Runtime.CoreCLR) 
+produced by the [coreclr repo](https://github.com/dotnet/coreclr). 
+
+Right in the description text there is an embedded git commit hash and lower 
+on the page is the Package History section with links to every build of the 
+package that has been published. For example the 
+[2.2.0-preview1-26608-04](https://dotnet.myget.org/feed/dotnet-core/package/nuget/runtime.win-x64.Microsoft.NETCore.Runtime.CoreCLR/2.2.0-preview1-26608-04) 
+build description says:
+
+    Internal implementation package not meant for direct consumption. Please do not reference directly. The .NET Core runtime,
+    called CoreCLR, and the base library, called System.Private.CoreLib. It includes the garbage collector, JIT compiler, base .NET
+    data types and many low-level classes. 311322beb96c5475fd7030fcd2f6e7ff14918853 When using NuGet 3.x this package
+    requires at least version 3.4.
```

Comment:
I think this would be better as a quote than as code, for readability. (Horizontal scrolling mainly.)